dot/core/service.go

```diff
@@ -555,3 +580,21 @@ func (s *Service) GetReadProofAt(block common.Hash, keys [][]byte) (
 
 	return block, proofForKeys, nil
 }
+
+// buildExternalTransaction builds an external transaction based on the current TransactionQueueAPIVersion
```

Comment:
Generally comments and logs should use explanatory names so, `TransactionQueueAPIVersion` would become say `transaction queue API version`.